### DISCUSSION TITLE
Manual rotation

### DISCUSSION THREAD

**Discussion by otisdog8** [jonaswinkler/paperless-ng#1280] [Feature Requests]
Sometimes the automatic rotation won't work, and it would be ideal if it was possible to, from the web UI, to rotate the document so that it is correct.

**Comment by monarc83** [comment]
Exactly the feature was looking for tonight. Would be great to have it on board ;-)

**Comment by zwischennetz** [comment]
+1

**Comment by thomassaffer** [comment]
+1 also from my side :-)

**Comment by HellmannK** [comment]
Hey! Is there any solution to rotate documents so far? I scanned all of my documents today, the first two were correct the rest is upside down... xD I dont see any chance to rotate them, or can I reran any command?

**Comment by leandrocoding** [comment]
This would be really useful!
Is this being worked on? 
Is it not possible? Too much work?
No interest?

**Comment by DevArounder** [comment]
It's the same with me, I've already demonized my scanner. It sends the data directly to paperless.

For the most part, the documents are the right way around. Sometimes, however, then the wrong way around, which is very annoying.
If you could turn off the function manually would be great.

**Comment by Linutux** [comment]
I would LOVE this feature. Sometimes the auto rotation in paperless doesn't work, so I need a way to correct it.

**Comment by xxLeoxx93** [comment]
Looking for thsi as well.

**Comment by thomas41587** [comment]
+1 for this feature!
Is there no way to do that (even manually?!)

**Comment by Schello805** [comment]
I also looking for a solution at the moment and can't find it :-(

**Comment by DarrenPIngram** [comment]
I can't say it is an easy solution to implement, or propose how to
programatically do it (or offer to do it), but it does not feel it should
be difficult to do.

I just wish I was a bit brighter with the old programming...
